tracee-ebpf/tracee/proc.go

```diff
@@ -0,0 +1,50 @@
+package tracee
+
+import (
+	"github.com/prometheus/procfs"
+)
+
+// gatherProcessTreeMap will put all PIDs into the returned map that are
+// children of filteredPID
+func gatherProcessTreeMap(filteredPID uint32) (map[uint32]uint32, error) {
+	fs, err := procfs.NewFS("/proc")
+	if err != nil {
+		return nil, err
+	}
+
+	procs, err := fs.AllProcs()
+	if err != nil {
+		return nil, err
+	}
+
+	filterPidAndChildPids := map[uint32]uint32{
+		filteredPID: 1,
```

Comment:
Is the value always set to 1?
You can use the value to know if we shuold trace the process or not (0 value to indicate tree!=x)